cabby/geo/directions.py

```diff
@@ -17,28 +17,30 @@
 
 from cabby.geo import util
 
+
 class Direction(enum.IntEnum):
-  """An enum to label several egocentric direction relations."""
-  AHEAD = 0
-  SLIGHT_LEFT = 1
-  LEFT = 2
-  SLIGHT_RIGHT = 3
-  RIGHT = 4
-  BEHIND = 5
+    """An enum to label several egocentric direction relations."""
+    AHEAD = 0
+    SLIGHT_LEFT = 1
+    LEFT = 2
+    SLIGHT_RIGHT = 3
+    RIGHT = 4
+    BEHIND = 5
```

Comment:
Is your tabbing set to four spaces? (We should be going in the other direction -- fixing places where we have 4 space tabs so that they are 2 spaces.)

Reply:
Now setting is set to 2.

Reply:
retrived old version